### PR TITLE
Add random dataset on fastfield bench.

### PR DESCRIPTION
**Pull request by fmassot**

### CODE REVIEW

**Review by PSeitz — APPROVED** [review]
Only 256 different values on purpose?

**Comment by fmassot** [comment]
I tried `u32` and `u64`.
If I remember correctly `u32` gives the same kind of results, `u64` is not chosen by the multilinear interpolation because it's not applicable.

File: fastfield_codecs/src/main.rs [hunk 1]
```diff
@@ -91,6 +91,10 @@ pub fn get_codec_test_data_sets() -> Vec<(Vec<u64>, &'static str)> {
         .collect::<Vec<_>>();
     data_and_names.push((data, "Almost monotonically increasing"));
 
+    let data = (1000..=200_000_u64)
+        .map(|_| rand::random::<u8>() as u64)
```
Comment (fulmicoton) [inline fastfield_codecs/src/main.rs:95]: Can we seed that?

**Review by fulmicoton — CHANGES_REQUESTED** [review]
I would rather seed that one to avoid extra noise in the bench.

File: fastfield_codecs/src/main.rs (hunk 1 above)
Comment (fmassot) [reply fastfield_codecs/src/main.rs:95]: I will. But the the results is already very stable on 200 000 values

**Comment by fmassot** [comment]
Closing this PR as it will be done in #1217

**Pull request closed without merging**